src/schema/users_model.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime, timezone
+
+from pydantic import BaseModel, EmailStr, Field, HttpUrl
+
+
+class lvls(BaseModel):
```

Comment:
Name should convey a meaning an explicit one 🫡 also maintain pascal convention for naming classes